### ISSUE TITLE
[DateRangePickers] Accessibility issues

### ISSUE THREAD

**Issue by filipKovachev** [telerik/kendo-react#1491]
In order for the DateRangePicker to be 508 accessibility compliant the user needs to be able to tab through the component via Tab, with the current implementation at some point after pressing Tab the focus gets lost and is put back to the browser itself. Any HTML elements below the DateRangePicker can never be reached using the tab key:

https://codesandbox.io/s/stupefied-chebyshev-h2obey

If the appendTo prop is set and the Popup is rendered  inside the DateRangePicker's child elements, then the tabbing order is correct, however this creates a second issue:

When using shift and tab to get back to the first input this is not possible, because focus cycles between the end date input and the calendar popup. This seems to be the default behavior and it can be seen in the following demo:

1. Click the "start date" input.
2. Press tab. focus goes to "end date" (correctly)
3. Press shift-tab. focus goes into the calendar controls (incorrectly. it should go back to start date input)
now keep pressing shift-tab and you are stuck in a loop between "end date" input and the calendar popup controls. You can't get back to the "start date"

https://stackblitz.com/edit/react-jcfp5c?file=app%2Fmain.jsx

Ticket ID: 1597095

**Comment by rally25rs** [comment]
I noticed something interesting when investigating this further;

The file `packages/popup/src/PopupWithoutContext.tsx` contains a conditional `this.props.appendTo !== null` that partially decides how/where to render the popup.
The default value for `appendTo` is `undefined` since the `DateRangePicker` does not pass anything, which goes to the false path of this conditional.

It turns out that just passing `null` makes the popup render in the right place in the DOM, because then the position becomes based on the popup's `trigger` instead of the root document.

So the workaround is to pass:
```
<DateRangePicker
  popupSettings={{
    appendTo: null,
  }}
/>
```

It seems like `DateRangePicker` should pass `appendTo: null` to `Popup` by default if not overridden in the props; or the conditional in `PopupWithoutContext` should check for `undefined or null` not just `null`.

**Comment by iadnonov** [comment]
Fixed in [v7.0.2-develop.3](https://www.telerik.com/kendo-react-ui-develop/components/changelogs/ui-for-react/#v7.0.2-develop.3)

**Comment by iadnonov** [comment]
Official version released v7.0.2
